JT/01_environment_setup/README.adoc

```diff
@@ -0,0 +1,406 @@
+
+* Git - Use Git if you need to work with version control and collaborate on code. It's a distributed version control system that helps track changes in source code during software development.
+
+=== Programming Languages SDK
+
+* Java SDK/Kotlin SDK - If you need to develop and test applications written in Java or Kotlin, use the Java SDK or Kotlin SDK, respectively. These Software Development Kits provide the necessary tools, libraries, and documentation for developing applications in these languages.
+
+=== Frontend Frameworks
+
+* Node.js/React/Angular/Vue.js - If you need to work with frontend development for web applications, use Node.js, React, Angular, or Vue.js. These tools and frameworks help in building scalable and efficient web applications with JavaScript.
+
+=== API Testing Tools
+
+* Postman/Insomnia/SoapUI/JMeter - If you need to test APIs, use Postman, Insomnia, SoapUI, or JMeter. These tools provide a user-friendly interface for sending requests, managing environments, and automating API tests.
+
+== [[software-installation]]Software Installation
+
+=== [[sdk-maven]]SDK, Maven
+
+Java SDK: The Java Software Development Kit (Java SDK), also known as the Java Development Kit (JDK), is a comprehensive suite of tools for developing Java applications. It includes the Java Runtime Environment (JRE), a compiler, a debugger, and other utilities.
+
+Maven: Apache Maven is a build automation and dependency management tool primarily used for Java projects. It simplifies the build process, manages project dependencies, and ensures consistent builds across different environments.
+
+. Download and install Java SE Development Kit and Maven
++
+image::resources/SDK-01.png[]
+
+In our case, this is version 21 as LTS, and we choose this one.
+
+After that, download and install Apache Maven or another type of building tool.
+
+image::resources/maven-01.png[]
+
+. Set up Java and Maven as Environment variables:
++
+Press [Win + Q] and type '_envir_' for example:
+
+image::resources/SDK-02.png[]
+
+Click on 'Edit environment variables for your account', choose the tab [Advanced] and click on the '**Environment Variables** ...' button:
+
+image::resources/SDK-03.png[]
+
+Here you will create System variable (or Edit them in case they already exist).
+- _JAVA_HOME_: add as value the path where you have installed Java 21 earlier (for example, _C:\Program Files\Java\jdk-21.0.2_)
+- _M2_HOME_: add as value the path where you have unzipped the archive earlier (for example, _C:\Program Files\apache-maven-3.6.3-bin_)
+
+image::resources/SDK-04.png[]
+
+Now look for the System variable called Path and edit its content. Add to this the following arguments if they do not already exist: _%M2_HOME%\bin; %JAVA_HOME%\bin_
+
+image::resources/SDK-05.png[]
+
+. Check to see that you have set up Java and Maven correctly:
+* Open a command line interface (CLI): (Press [Win + R] and type '_cmd_' and press 'Enter')
+image::resources/SDK-06.png[]
```

Comment:
Please check if image displaying correctly in .adoc format

Reply:
Thank you for the careful reading of the document and the very accurate checking of all the details. Now I know whom I can trust with the review of my main project :))
And I'm sure that at least once my README was read in its entirety :)